src/script/vco-ie/main_warhammer/wh3_main_ksl_the_great_orthodoxy/missions.lua

```diff
@@ -0,0 +1,185 @@
+local missions = {
+	[[
+ 		mission
+		{
+			victory_type vco_victory_type_ksl_ort_1_there_can_only_be_one;
+			key wh_main_short_victory;
+			issuer CLAN_ELDERS;
+			primary_objectives_and_payload
+			{
+
+				objective
+				{
+					override_text mission_text_text_vco_ksl_ort_1_there_can_only_be_one;
+					type SCRIPTED;
+					script_key vco_dummy;
+				}
+
+				objective
+				{
+					type HAVE_AT_LEAST_X_OF_A_POOLED_RESOURCE;
+					total 600;
+					
+					pooled_resource wh3_main_ksl_followers;
+				}
+			
+				objective
+				{
+					type CONTROL_N_REGIONS_INCLUDING;
+					total 3;
+				
+					region wh3_main_combi_region_erengrad;
+					region wh3_main_combi_region_praag;
+					region wh3_main_combi_region_kislev;
+				}
+
+				payload
+				{
+					effect_bundle
+					{
+						bundle_key vco_victory_payload_ksl_ort_1_there_can_only_be_one;
+						turns 0;
+					}
+
+					effect_bundle
+					{
+						bundle_key vco_campaign_victory_dummy;
+						turns 0;
+					}
+
+					game_victory;
+				}
+			}
+		}
+	]],
+	[[
+ 		mission
+		{
+			victory_type vco_victory_type_ksl_ort_2_the_last_bastion;
+			key wh_main_short_victory;
+			issuer CLAN_ELDERS;
+			primary_objectives_and_payload
+			{
+				objective
+				{
+					override_text mission_text_text_vco_ksl_ort_2_the_last_bastion;

```

Comment:
I think you can also merge these two. Not 100% sure but I think Destroy Faction accepts overiden if there is more than one key. Also add the confederation_valid flag.